src/routes/tools/comparative-slr/_Title.svelte

```diff
@@ -0,0 +1,66 @@
+<script>
+  import { Button, InlineNotification } from "carbon-components-svelte";
+  import { Location16 } from "carbon-icons-svelte";
+  import { getCSSProp } from "~/helpers/utilities";
+
+  export let location;
+  export let loadLocation;
+  export let floodScenario;
+  export let timeFrame;
+  export let dataLayers;
+  export let dataUnavailableMsg = "";
+
+  $: visibleLayers = dataLayers.filter((d) => d.checked);
+  $: layerHtml = getLayerHtml(visibleLayers);
+
+  const getLayerHtml = (layers) => {
+    const html = layers.map(
+      ({ label, color }) =>
+        `<span class="annotate annotate-color" style="background:${getCSSProp(
```

Comment:
We don't do this in any of the other tools, but it seems like it could be useful here to help reaffirm which colors correspond to which data sources. Let me know what you think @mukhtyar

Reply:
Yes, for sure. It could also be used to call out GCMs.